src/dialects/DialectPostgres.ts

```diff
@@ -296,6 +296,11 @@ export class DialectPostgres extends Dialect {
                 indices: [],
                 comment: column.description ?? undefined,
             };
+            if (column.column_default) {
+                if (column.column_default.match(/^[a-z_0-9]+\(.{0,}?\)$/)) {
```

Comment:
This type of regex will exclude default values like `9.99` or `CURRENT_TIMESTAMP`.
I think we can remove the regex condition and apply `Sequelize.literal` to all types of default values like so:

```ts
if (column.column_default) {                
    columnMetadata.defaultValue = `Sequelize.literal("${column.column_default}")`;
}
```
On top of my head this should handle the following cases:
- primitive values like `'abc'` or `123`
- constants like `CURRENT_TIMESTAMP`
- functions like `nextval(...)`

Reply:
removed